content/faq/do-i-need-a-credit-card/index.md

```diff
@@ -12,6 +12,6 @@ resources:
   - src:
 ---
 
-No credit card is needed to sign up for either the BluBracket Community Edition or free trial.
```

Comment:
All mention of "free trial" is being removed or clarified. BluBracket Community Edition is always free, and the trial of BluBracket Teams is not time-limited.